### PR TITLE
[Price Impact] Confirmation Modal Warnings

### PR DESCRIPTION
**Pull request by W3stside**
Part of #1456 

Merges into #1544 

Adds warnings into TX Confirmation modal

### CODE REVIEW

File: src/components/TradeWidget/TxMessage.tsx [hunk 1]
```diff
@@ -258,6 +260,75 @@ export const TxMessage: React.FC<TxMessageProps> = ({ sellToken, receiveToken, n
...
+    amount: sellTokenAmount,
+  })
+
+  const { priceImpactWarning } = usePriceImpact({
+    ...baseProps,
+    limitPrice: price,
+    fillPrice,
+  })
+
+  logDebug('[TX-MESSAGE] limit price:', sellTokenAmount)
```
Comment (W3stside) [inline src/components/TradeWidget/TxMessage.tsx:282]: remove these bad boys

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr1550--dexreact.review.gnosisdev.com)**: Testing web app
   * **📚 [Storybook](https://pr1550--dexreact.review.gnosisdev.com/storybook/)**: Component stories

File: src/components/trade/PriceImpact/utils.ts [hunk 2]
```diff
@@ -21,6 +20,13 @@ enum ImpactLevel {
   NONE = 0,
 }
 
+const WARNINGS = {
```
Comment (anxolin) [inline src/components/trade/PriceImpact/utils.ts:23]: good call

**Review by anxolin — COMMENTED** [review]
1) It doesn't show the Price Impact together with the main information. My understanding is that it should be between the price and validity dates.

2) I like that you added a title to the WARNING ("Price Alert"). I think we should do the same in the other warning for coherence. Something like "Minimal Trade Alert"

Other than that, looks great

**Comment by W3stside** [comment]
> 1. It doesn't show the Price Impact together with the main information. My understanding is that it should be between the price and validity dates.
> 2. I like that you added a title to the WARNING ("Price Alert"). I think we should do the same in the other warning for coherence. Something like "Minimal Trade Alert"
> 
> Other than that, looks great

What do you mean in point 1? You want a whole other section for it?

**Comment by anxolin** [comment]
> What do you mean in point 1? You want a whole other section for it?

Similarly as Uniswap does:
image

I mean specifically:
image

**Review by anxolin — DISMISSED** [review]
The impact looks nice in the confirmation modal.

However, it looks like the issue you found is still there
image

**Comment by W3stside** [comment]
@anxolin made the incorrect warnings inside another pr, #1557

**Review by anxolin — COMMENTED** [review]
Let's consolidate all the other fixes into the final release, and test all together

**Pull request merged**